app/forms/support_user_invite_form.rb

```diff
@@ -0,0 +1,51 @@
+class SupportUserInviteForm < ApplicationForm
+  FORM_PARAMS = %i[first_name last_name email].freeze
+
+  attr_accessor :email, :first_name, :last_name, :service
+
+  validate :validate_support_user
+  validates :service, presence: true
+
+  def invite!
+    save_support_user
+  end
```

Comment:
I realise there's some additional work needed to raise an error if persistence fails when the form object is valid, but let's build upon that foundation.

This should be `persist` and the caller can call either `save` or `save!`.

```suggestion
  def persist
    save_support_user
  end
```

Reply:
resolved in [ac16cba](https://github.com/DFE-Digital/itt-mentor-services/pull/253/commits/ac16cba008ff65bed305d3cff34dce4ba115f505)